rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/run/configurations/unityExe/UnityExeConfigurationType.kt

```diff
@@ -0,0 +1,17 @@
+package com.jetbrains.rider.plugins.unity.run.configurations.unityExe
+
+import com.intellij.execution.configurations.ConfigurationTypeBase
+import com.intellij.util.IconUtil
+import icons.UnityIcons
+
+class UnityExeConfigurationType : ConfigurationTypeBase("RunUnityExe", "Standalone Player", // "RunUnityExe" preserved for compatibility
+    "Unity Standalone Player configuration", IconUtil.scale(UnityIcons.RunConfigurations.UnityExe, null, 0.33f)) {
```

Comment:
Does this need to be scaled? We're not scaling the other icons

Reply:
omg, I missed that.